hack/run.sh

```diff
@@ -37,6 +37,7 @@ function usage() {
   echo "   build-for-source-from-source                            Build artifacts from source for source bundle only"
   echo "   benchmark-filter <bencmark_class_name>                  Run the filter benchmarks for <benchmark_class_name>"
   echo "   benchmark-filters                                       Run all the filter benchmarks"
+  echo "   run-matched-test <testname> <testdir>                   Run all e2e tests matching testname in testdir"
```

Comment:
Why not calling it `run-e2e-test <testname> <testdir>` ?

Reply:
Good point, that makes more sense for sure - I'll update this